projects/backend/mockgenerator/src/generators/cms/components.ts

```diff
@@ -83,7 +83,7 @@ export class ComponentGenerator extends ClientGenerator {
             const idList = JSON.parse(response.bodyAsText);
 
             // this.getChildComponents(pageData).forEach(uid => {
```

Comment:
Since it's a commented out code, let's just remove it?

Reply:
I took this into account, but idk destination of this code. Kacper proposed to leave it.